### DISCUSSION TITLE
Bad site look on iPhone 5

### DISCUSSION THREAD

**Discussion by Alexey-T** [community/community#55645] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I see bad look of GitHub site for few days. On iPhone 5 SE. Initially site shows ok, after 1-3 seconds site becomes scaled so that site occupies only 40…50 percents of the iPhone width. I must zoom it in.

**Comment by scaomath** [comment]
I experienced something similar.

![Screenshot_20230513-071537](https://github.com/community/community/assets/4467042/90387924-c6b3-4ca9-b370-298f79cfa291)
